frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineVisualizationTask.tsx

```diff
@@ -180,18 +180,24 @@ const TaskComponent: React.FC<TaskProps> = ({
         {truncatedVisualName}
       </text>
       {isPipelineRun && showStatusState && (
-        <svg
-          width={30}
-          height={30}
-          viewBox="-5 -5 20 20"
-          style={{
-            color: status
-              ? getRunStatusColor(status.reason, t).pftoken.value
-              : getRunStatusColor(runStatus.Cancelled, t).pftoken.value,
-          }}
+        <g
+          className={cx({
+            'fa-spin odc-pipeline-vis-task--icon-spin': status.reason === runStatus.Running,
+          })}
         >
-          <StatusIcon status={status.reason} shiftOrigin />
-        </svg>
+          <svg
+            width={30}
+            height={30}
+            viewBox="-5 -5 20 20"
```

Comment:
```suggestion
            viewBox="-5 -4 20 20"
```
The icon sits a bit too low, let's bring it more to the center of the node.

Reply:
Updated. Had to also change `transform-origin` to re center the icon.

Reply:
transform-origin could be more into the direction of `16.7px 15.1px` . But don't want block the PR.

Still a confusing solution we have here. :confused:

Reply:
Updated.